crates/hdk/src/hash_path/path.rs

```diff
@@ -13,22 +13,28 @@ use validate::RequiredValidationType;
 /// See `impl From<String> for Path` below.
 pub const DELIMITER: &str = ".";
 
-/// "hdk.path" as utf8 bytes
 /// All paths use the same link tag and entry def id.
 /// Different pathing schemes/systems/implementations should namespace themselves by their path
 /// components rather than trying to layer different link namespaces over the same path components.
 /// Similarly there is no need to define different entry types for different pathing strategies.
-///
-/// @todo - revisit whether there is a need/use-case for different link tags or entries
-/// See anchors implementation.
-pub const NAME: [u8; 8] = [0x68, 0x64, 0x6b, 0x2e, 0x70, 0x61, 0x74, 0x68];
+/// The DHT_PREFIX ends up as both the prefix of the link tags and also in the
+/// `PathEntry` struct itself to mitigate collisions on the DHT.
+pub const DHT_PREFIX: [u8; 1] = [0x00];
```

Comment:
This one-byte prefix still feels kinda peculiar, but I get the purpose.

It also seems worth taking it out of the array, since the whole point is to streamline PathEntry, even adding a couple extraneous bytes to a 40 byte payload is 5% bloat.